packages/cashier/src/pages/account-transfer/account-transfer.tsx

```diff
@@ -1,17 +1,41 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Loading } from '@deriv/components';
 import { WS } from '@deriv/shared';
 import { connect } from 'Stores/connect';
-import AccountTransferNoAccount from './account-transfer-no-account';
+import RootStore from 'Stores/types';
 import Error from 'Components/error';
 import NoBalance from 'Components/no-balance';
 import { Virtual } from 'Components/cashier-container';
 import CashierLocked from 'Components/cashier-locked';
-import AccountTransferLocked from './account-transfer-locked';
-import AccountTransferForm from './account-transfer-form';
-import AccountTransferReceipt from './account-transfer-receipt';
 import CryptoTransactionsHistory from 'Components/crypto-transactions-history';
+import AccountTransferReceipt from './account-transfer-receipt';
+import AccountTransferForm from './account-transfer-form';
+import AccountTransferNoAccount from './account-transfer-no-account';
+import AccountTransferLocked from './account-transfer-locked';
+
+type TAccountTransfer = {
+    accounts_list: Array<object>;
+    container: string;
+    error: {
+        is_show_full_page: boolean;
+        message: string;
+    };
+    has_no_account: boolean;
+    has_no_accounts_balance: boolean;
+    is_cashier_locked: boolean;
+    is_crypto_transactions_visible: boolean;
+    is_loading: boolean;
+    is_switching: boolean;
+    is_transfer_confirm: boolean;
+    is_transfer_locked: boolean;
+    is_virtual: boolean;
+    onMount: () => void;
+    recentTransactionOnMount: () => void;
+    setAccountTransferAmount: (amount: number | string) => void;
+    setActiveTab: (container: string) => void;
+    setIsTransferConfirm: (status: boolean) => void;
+    setSideNotes: (notes: Array<object | string> | null) => void;
```

Comment:
Why do you use different types for the same things?

Reply:
Fixed.